src/main/java/no/ndla/taxonomy/rest/v1/CrudController.java

```diff
@@ -64,7 +65,7 @@ protected T doPut(URI id, UpdatableDto<T> command) {
         command.apply(entity);
 
         if (entity instanceof EntityWithPath && cachedUrlUpdaterService != null) {
-            cachedUrlUpdaterService.updateCachedUrls((EntityWithPath) entity);
+            cachedUrlUpdaterService.updateCachedUrls((Node) entity);
```

Comment:
Vi har behov for å bruke denne controlleren for nodeconnection, i forbindelse med å bygge programområder.

Reply:
Mener du at jeg skal legge det til i denne pr'en?

`NodeConnection` har vel aldri vært `EntityWithPath`?

Reply:
Nei, treng ikkje det. Men då må vi lage en egen pr på det etterpå.